telematic_system/telematic_apps/web_app/client/src/tests/api/api-dashboards.test.js

```diff
@@ -0,0 +1,47 @@
+import { afterAll, afterEach, beforeAll, beforeEach, expect, jest, test } from '@jest/globals';
+import { searchDashboards, listEventDashboards, updateEventDashboards, getDashboardsByOrg, deleteEventDashboards } from '../../api/api-dashboards';
+import axios from 'axios';
+
+jest.mock('axios');
+
+beforeEach(() => {
+    const response = { data: { status: 'success' } };
+    axios.get.mockResolvedValue(response);
+    axios.post.mockResolvedValue(response);
+    axios.delete.mockResolvedValue(response);
+})
+
+test('Search dashboard not throw', async () => {
```

Comment:
I'm not sure how much effort it will be, but it seems like we can increase the line coverage a fair bit for the client by inverting these tests to check for failure conditions as well

Reply:
It is already inverted.